ui/components/ui/list-item/list-item.component.js

```diff
@@ -19,6 +19,8 @@ export default function ListItem({
     className,
     subtitle || children ? '' : 'list-item--single-content-row',
   );
+  const showBorderTop =
```

Comment:
What if instead we add to the primaryClassName?

```
const primaryClassName = classnames(
    'list-item',
    className,
    subtitle || children ? '' : 'list-item--single-content-row',
    { 'list-item-border': process.env.MULTICHAIN && className === 'transaction-list-item'  }
  );
```

Reply:
And better yet, can we make this a prop instead of checking `className === 'transaction-list-item'` since we're passing `transaction-list-item` in when it's used.

Reply:
Nice suggestion 😄  Updated here https://github.com/MetaMask/metamask-extension/pull/19176/commits/3dc160b178053dfda4f054a475b10d67e7e5240d